index.bs

```diff
@@ -2019,23 +2025,29 @@ interface XRInputSourceEvent : Event {
   constructor(DOMString type, XRInputSourceEventInit eventInitDict);
   [SameObject] readonly attribute XRFrame frame;
   [SameObject] readonly attribute XRInputSource inputSource;
+  readonly attribute double? x;
+  readonly attribute double? y;
 };
 
 dictionary XRInputSourceEventInit : EventInit {
   required XRFrame frame;
   required XRInputSource inputSource;
+  double x;
+  double y;
 };
 </pre>
 
 The <dfn attribute for="XRInputSourceEvent">inputSource</dfn> attribute indicates the {{XRInputSource}} that generated this event.
 
 The <dfn attribute for="XRInputSourceEvent">frame</dfn> attribute is an {{XRFrame}} that corresponds with the time that the event took place. It may represent historical data. Any {{XRViewerPose}} queried from the {{XRInputSourceEvent/frame}} MUST have an empty {{XRViewerPose/views}} array.
 
+The <dfn attribute for="XRInputSourceEvent">x</dfn> and <dfn attribute for="XRInputSourceEvent">y</dfn> attributes define screen coordinates for events with {{XRInputSourceEvent/inputSource}}s that have a {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/"screen"}}. They are <code>null</code> for other kinds of events.
```

Comment:
Is "screen coordinates" the right term to use? Perhaps "viewport-relative coordinates"? Except it's possible for the viewports to be different in multi-view situations (I can't think of a screen-space multi-view situation, though).

Reply:
Also, should we be allowing this for other target ray modes? This may potentially be useful for the layers spec as well cc @cabanier

Reply:
You could have stereo screens via shutter/polarized glasses or lenticular displays. You'd still be interacting on the screen surface and projecting inward, though, so we should always have a single point of interaction on the screen surface.